theme_loftspace/views/snippets/s_carousel.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_carousel" inherit_id="website.s_carousel">
+    <xpath expr="//div[hasclass('carousel-item')]" position="attributes">
+        <attribute name="class" add="o_cc2" remove="o_cc5" separator=" "/>
```

Comment:
I propose `o_cc4` 😉

| this PR | proposal |
|--------|--------|
| <img width="1366" alt="image" src="https://github.com/user-attachments/assets/08aead00-170d-4102-9933-bccaa746a51d"> | <img width="1055" alt="image" src="https://github.com/user-attachments/assets/f9a5e4c3-89e9-4e7a-89e8-d24f62c95f57"> |